services/director-v2/tests/unit/conftest.py

```diff
@@ -0,0 +1,82 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+import pytest
+from models_library.service_settings_labels import SimcoreServiceLabels
+from simcore_service_director_v2.models.domains.dynamic_services import (
+    DynamicServiceCreate,
+)
+from simcore_service_director_v2.models.schemas.dynamic_services import (
+    MonitorData,
+    ServiceDetails,
+    ServiceLabelsStoredData,
+)
+
+
+@pytest.fixture
+def simcore_services_network_name() -> str:
+    return "test_network_name"
+
+
+@pytest.fixture(autouse=True)
+def disable_dynamic_sidecar_monitor_in_unit_tests(
+    monkeypatch, simcore_services_network_name: str
+) -> None:
+    monkeypatch.setenv("DIRECTOR_V2_DYNAMIC_SERVICES_enabled", "false")
+    monkeypatch.setenv("REGISTRY_auth", "false")
+    monkeypatch.setenv("REGISTRY_user", "test")
+    monkeypatch.setenv("REGISTRY_PW", "test")
+    monkeypatch.setenv("REGISTRY_ssl", "false")
+    monkeypatch.setenv("SIMCORE_SERVICES_NETWORK_NAME", simcore_services_network_name)
+    monkeypatch.setenv("TRAEFIK_SIMCORE_ZONE", "test_traefik_zone")
+    monkeypatch.setenv("SWARM_STACK_NAME", "test_swarm_name")
+
+
+@pytest.fixture
+def simcore_service_labels() -> SimcoreServiceLabels:
+    return SimcoreServiceLabels(
+        **SimcoreServiceLabels.Config.schema_extra["examples"][1]
+    )
+
+
+@pytest.fixture
+def dynamic_service_create() -> DynamicServiceCreate:
+    return DynamicServiceCreate.parse_obj(ServiceDetails.Config.schema_extra["example"])
+
+
+@pytest.fixture
+def service_labels_stored_data() -> ServiceLabelsStoredData:
+    return ServiceLabelsStoredData.parse_obj(
+        ServiceLabelsStoredData.Config.schema_extra["example"]
+    )
+
+
+@pytest.fixture
+def dynamic_sidecar_port() -> int:
+    return 1222
+
+
+@pytest.fixture
+def monitor_data_from_http_request(
+    dynamic_service_create: DynamicServiceCreate,
+    simcore_service_labels: SimcoreServiceLabels,
+    dynamic_sidecar_port: int,
+) -> MonitorData:
+    return MonitorData.from_http_request(
+        service=dynamic_service_create,
+        simcore_service_labels=simcore_service_labels,
+        port=dynamic_sidecar_port,
+    )
+
+
+@pytest.fixture
+def monitor_data_from_service_labels_stored_data(
+    service_labels_stored_data: ServiceLabelsStoredData, dynamic_sidecar_port: int
+) -> MonitorData:
+    return MonitorData.from_service_labels_stored_data(
+        service_labels_stored_data=service_labels_stored_data, port=dynamic_sidecar_port
+    )
+
+
+@pytest.fixture
+def ensure_in_docker_swarm(docker_swarm: None) -> None:
+    pass
```

Comment:
why do you rename that fixture? it confuses me greatly.
Also why not enhancing the docker swarm in pytest-simcore instead?

Reply:
I've removed it as it was not required.